VarLife.md

```diff
@@ -0,0 +1,59 @@
+## OTCA Metapixel
+
+<img src="http://www.conwaylife.com/w/images/3/3a/Otcametapixel.png"/>
+(Source: [http://www.conwaylife.com/w/images/3/3a/Otcametapixel.png](http://www.conwaylife.com/w/images/3/3a/Otcametapixel.png))
+
+The [OTCA Metapixel](http://www.conwaylife.com/w/index.php?title=OTCA_metapixel) is a construct in Conway's Game of Life that can be used to simulate any Life-like cellular automata. As the LifeWiki (linked above) says,
+
+> The OTCA metapixel is a 2048 × 2048 period 35328 unit cell that was constructed by Brice Due... It has many advantages...including the ability to emulate any Life-like cellular automaton and the fact that, when zoomed out, the ON and OFF cells are easy to distinguish...
+
+What [Life-like cellular automata](http://www.conwaylife.com/wiki/Life-like_cellular_automaton#Life-like_cellular_automata) means here is essentially that cells are born and cells survive according to how many of their eight neighbor cells are alive. The syntax for these rules is as follows: a B followed by the numbers of live neighbors that will cause a birth, then a slash, then an S followed by the numbers of live neighbors that will keep the cell alive. A bit wordy, so I think an example will help. The canonical Game of Life can be represented by the rule B23/S3, which says that any dead cell with two or three live neighbors will become alive and any live cell with three neighbors will remain alive. Otherwise, the cell dies.
+
```

Comment:
You might want to go a bit more in-depth on the OTCA metapixel here - the overlap for adjacent metapixels, how the rules are set, how neighbor detection works, etc.

Reply:
Those details are important in the generation of GoL patterns from Varlife patterns, so they are relevant to our work. I'm not suggesting an essay on them - a summary will be sufficient.